chanacceptor/interface.go

```diff
@@ -33,18 +34,58 @@ type ChannelAcceptResponse struct {
 	// ChanAcceptError the error returned by the channel acceptor. If the
 	// channel was accepted, this value will be nil.
 	ChanAcceptError
+
+	// UpfrontShutdown is the address that we will set as our upfront
+	// shutdown address if the peer supports the feature bit.
+	UpfrontShutdown lnwire.DeliveryAddress
```

Comment:
Generally a bit unsure of this type of comment, because it's easy for the merge function to up renamed/changed without the comment being updated, and ideally people who add new fields will unit test? Not strong feelings, so added it, but more of a meta thought.

Reply:
I think commenting probably doesn't hurt in this case as the current unit test would miss added fields.